android-components/components/lib/crash/build.gradle

```diff
@@ -90,7 +90,6 @@ dependencies {
     testImplementation ComponentsDependencies.mozilla_glean_forUnitTests
 }
 
-ext.gleanGenerateMarkdownDocs = true
```

Comment:
@travis79 I am wondering if we can remove their respective generated metrics.md file as well? 
https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/lib/crash/docs/metrics.md
https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/service/nimbus/docs/metrics.md

Reply:
You sure can! The Glean Dictionary now serves this purpose of documentation of our telemetry collections, so the local file that was generated is no longer needed. In fact, it's probably good to remove it just because it will soon be out of sync since the generation step for it is being removed.